common/models/model-method.js

```diff
@@ -2,6 +2,8 @@
 // Node module: loopback-workspace
 // This file is licensed under the MIT License.
 // License text available at https://opensource.org/licenses/MIT
+var app = require('../../server/server');
```

Comment:
Models in "common" should not depend on things in "server", that goes against the direction of dependencies. You can access the app via `ModelMethod.app` after the model was configured (attached to a datasource & app).